### ISSUE TITLE
view `folder_contents` broken if contains any item with `image` field filled

### ISSUE THREAD

**Issue by sauzher** [isawnyu/isaw.web#565]
There is a Plone bug I just discovered  (and reproduced even on classic.demo.plone.org)
Long story short: we cannot have a catalog metadata column named like any dexterity blob field.

`isaw.policy` adds up a portal_catalog column metadata called `image`; this name matches some named blob imaged fields no matter the portal_type (Image, exibitions... )

This breaks folder_contents view if it contains items  implementing a populated blob field named `image`. More precisely it'll break the items json representation in the @@getVocabulary call:

This is the python repr: the image field contains an object no json serializable
```
{'CreationDate': '2024-12-06T11:12:43-05:00',
[ ... ]
 'getMimeIcon': '/isawnotheme/++resource++mimetype.icons/txt.png',
 'getObjSize': '890.0 KB',
 'getURL': 'http://localhost:8081/isawnotheme/esibizione-1',
'id': 'esibizione-1',

 'image': <plone.namedfile.file.NamedBlobImage object at 0x7f5babcf17d0>,

[ ... ]

 'total_comments': 0}
```

Brute solution to me, is to remove metadata "image" from the catalog. It seems that it's never used/accessed in the code, It only appears as old [ATCT Topic columns configuration](https://github.com/isawnyu/isaw.web/blob/8d6fda8ed78bf6ad445876eaf360ed7a28a30408/src/isaw.policy/src/isaw/policy/profiles/default/portal_atct.xml#L65) and so, there might be some Topic/Collection around configured to show the "image" field in topic tabular view.

@alecpm What do you think?

**Comment by sauzher** [comment]
ref. https://github.com/plone/Products.CMFPlone/issues/4078

**Comment by alecpm** [comment]
Removing that column seems like a very good idea. Alternatively we could register an indexed for it that behaved in whatever way the code intended that column to behave (which probably wasn’t storing the blob (or a reference to it) in the catalog.

**Comment by sauzher** [comment]
Good point. I'd say, we go for removal today. if tomorrow the column will be necessary for any Collection we'll implement the index.

**Comment by paregorios** [comment]
@sauzher was this change completed, or is it still awaiting work? If it's completed, can we close the ticket?
